po/tr.po

```diff
@@ -1303,9 +1302,8 @@ msgid "Go to a random unread article"
 msgstr "Rastgele okunmamış bir yazıya git"
 
 #: src/keymap.cpp:126
-#, fuzzy
 msgid "Open URL of article, or entry in URL view. Mark read"
-msgstr "Yazının URL'sini veya URL görünümündeki girdiyi aç. Okundu imle."
+msgstr "Yazının URL'sini veya URL görünümündeki girdisini aç. Okundu imle."
```

Comment:
None of the help messages end with a period, so please remove it here.

Reply:
Fixed.